tests/pkg-unique-id_duplicate/EPUB/package.opf

```diff
@@ -11,17 +10,17 @@
     <dc:identifier id="pub-id">pkg-unique-id</dc:identifier>
     <dc:language>en</dc:language>
     <dc:publisher>W3C</dc:publisher>
-    <dc:title>Unique identifier reused</dc:title>
+    <dc:title>pkg-unique-id</dc:title>
```

Comment:
pkg-unique-id_duplicate?

Reply:
Ouch... I did not realize this test and I missed this... Thanks for noting this, @davemanhall!